spec/system/search_results_page_spec.rb

```diff
@@ -50,37 +50,31 @@
     end
   end
 
-  # rubocop:disable RSpec/ExampleLength
   describe "facets" do
     it "shows expected facets" do
       visit '/?search_field=all_fields&q='
       domain_facet_html = '<div class="card facet-limit blacklight-domain_ssim ">'
       expect(page.html.include?(domain_facet_html)).to be true
 
-      community_facet_html = '<div class="card facet-limit blacklight-community_root_name_ssi ">'
+      community_facet_html = '<div class="card facet-limit blacklight-communities_ssim ">'
       expect(page.html.include?(community_facet_html)).to be true
 
       type_facet_html = '<div class="card facet-limit blacklight-genre_ssim ">'
       expect(page.html.include?(type_facet_html)).to be true
 
       year_facet_html = '<div class="card facet-limit blacklight-year_available_itsi ">'
       expect(page.html.include?(year_facet_html)).to be true
-
-      # Collection facet is not rendered by defailt
-      collection_facet_html = '<div class="card facet-limit blacklight-collection_name_ssi ">'
-      expect(page.html.include?(collection_facet_html)).to be false
```

Comment:
Now this field is rendered by default because the PDC Describe records give it a new meaning.